samples/java_springboot/50.teams-messaging-extensions-search/README.md

```diff
@@ -17,35 +17,19 @@ build a Search-based Messaging Extension.
 > Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
 the Teams service needs to call into the bot.
 
-1) Clone the repository
-
-    ```bash
-    git clone https://github.com/Microsoft/botbuilder-java.git
-    ```
-
-1) Run ngrok - point to port 3978
-
-    ```bash
-    ngrok http -host-header=rewrite 3978
-    ```
-
-1) Create [Bot Framework registration resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration) in Azure
+- Create [Bot Framework registration resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration) in Azure
```

Comment:
In this step I added a reference to the deploy section, I don't know if it is correct